### DISCUSSION TITLE
Layer not working

### DISCUSSION THREAD

**Discussion by ArshathAli** [komoot/photon#809] [Q&A]
I have installed OSM data for India using  Nominatim and then imported it to photon. When I try to do a request using 
**/api?q=Ooty&layer=city** . (For ref: Ooty is a small city/town in the South of India).
then I get empty response like this  **{"features":[],"type":"FeatureCollection"}**  else I get the required values.  
I have installed nominatim with full setting.

Can anybody help me out. Thanks in advance

**Comment by lonvia** [comment]
Do you find the town of "Ooty" in your Nominatim installation? Do you find "Ooty" in Photon when you don't use the layer parameter? If yes, what is the result? What versions of Nominatim and Photon are you using?
